### ISSUE TITLE
Tables print with middle lines

### ISSUE THREAD

**Issue by abreis** [Nukesor/comfy-table#43]
Using `presets::ASCII_BORDERS_ONLY`, whose sample table is:

```text
+---------------+
| Hello   there |
+===============+
| a       b     |
| c       d     |
+---------------+
```

I'm instead getting an empty middle row between each actual row:

```text
+-------------------------------+
| timestamp   one   two   three |
+===============================+
| 2001000     12    15    19    |
|                               |
| 2001001     22    25    29    |
|                               |
| 2001002     32    35    39    |
+-------------------------------+
```

This is with v4.0.1. Is this a bug? Or some row padding section I'm not seeing? 

Thanks!

**Comment by abreis** [comment]
Same with `presets::ASCII_HORIZONTAL_BORDERS_ONLY`. 

Sample:

```text
---------------
 Hello   there
===============
 a       b
 c       d
---------------
```

Actual output:
```text
-------------------------------
 timestamp   one   two   three 
===============================
 2001000     12    15    19    
-------------------------------
 2001001     22    25    29    
-------------------------------
 2001002     32    35    39    
-------------------------------
```

**Comment by Nukesor** [comment]
Ok.

The first one actually seems to be broken. The newline in between the horizontal rows isn't supposed to be there.

However, in the `presets::ASCII_HORIZONTAL_BORDERS_ONLY`, the sample is actually wrong.
Looking at it, the naming for those presets is kind of confusing and probably not accurate.

I guess the `BORDERS` part could simply be removed from the name for that preset?

**Comment by abreis** [comment]
Ah, right, the `ASCII_HORIZONTAL_BORDERS_ONLY` is effectively `ASCII_MARKDOWN` (no lines between rows, no surrounding borders).

Yes, I think removing 'BORDERS' would be enough for that preset, plus fixing the sample (they're really useful!). That just leaves finding out what's causing the newlines in the first example.

**Comment by Nukesor** [comment]
Ok. The preset samples were indeed a little imprecise.

I rewrote the tests to actually use the sample text from the `style/presets.rs` module. This should make things a little safer and more consistent.
Furthermore, I'll keep the `presets::ASCII_BORDERS_ONLY` as it is for now, the preset sample should soon be correct.
The current preset is probably used by quite a few people and this would break backward compatibility.
This preset is also much easier to read. Without horizontal lines, it's super hard to differentiate between rows.

Anyhow, I'm open for a new preset!
I don't have a good name for it yet.

```
  +/// Just like ASCII_BORDERS_ONLY, but without horizontal middle lines.
  +///
  +/// ```text
  +/// +---------------+
  +/// | Hello   there |
  +/// +===============+
  +/// | a       b     |
  +/// | c       d     |
  +/// +---------------+
  +/// ```
  +pub const ASCII_BORDERS_ONLY_WITHOUT_HORIZONTAL_DELIMITER: &str = "||--+==+     --++++";
  ```
  
  In case you don't need it to be added to the crate, you can just take the preset string from above and import it in your table like this:
```
table.load_preset("||--+==+     --++++");

**Comment by abreis** [comment]
I agree with not breaking horizontal compatibility, and the current presets are fine as they are.

I do think a new preset would be nice to have. Although harder to read, more compact presets are very useful when one expects to print many rows (50% fewer scrolling!).

As for naming, my suggestion would be to use `COMPACT` or `CONDENSED`, e.g. `ASCII_BORDERS_ONLY_CONDENSED`.

Thanks again!

**Comment by Nukesor** [comment]
I like the `CONDENSED` name.
Implemented in 2f7a1ea. I'll soon release a new minor version :)
